client/src/client.rs

```diff
@@ -121,9 +121,6 @@ fn handle_defaults<'a, 'b>(
         let defaults_i = defaults.len() - 1 - i;
         if args[args_i] == serde_json::Value::Null {
             if first_non_null_optional_idx.is_some() {
-                if defaults[defaults_i] == serde_json::Value::Null {
-                    panic!("Missing `default` for argument idx {}", args_i);
-                }
```

Comment:
What is the reason that this is this removed?

Reply:
Hey, thanks for reviewing the PR :) I removed this so that we can pass `null()` to the rpc commands and therefore make the node use the default configuration created internally by the node.

Reply:
I'm only newish to this crate but I'm almost certain we cannot remove this check, doing so changes the behaviour of the whole crate.